Libs/DICOM/Core/ctkDICOMDatabase.h

```diff
@@ -430,6 +430,15 @@ class CTK_DICOM_CORE_EXPORT ctkDICOMDatabase : public QObject
   ///  - instanceUID (unique)
   void instanceAdded(QString);
 
+  /// This signal is emitted when a patient is removed from CTK database.
```

Comment:
This signal will not be triggered when a patient is removed from the database. It will only be triggered if it is deleted by calling `removePatient` method on this object. Unless it is an in-memory database, the database can be simultaneously accessed from via several ctkDICOMDatabase objects in the same process or other processes, so not all deletions will trigger these signals. Update the method documentation to make this clear.

Reply:
Modifications done